dalle_flow_glid3/sample.py

```diff
@@ -95,29 +98,41 @@ def set_requires_grad(model, value):
 bert.half().eval()
 set_requires_grad(bert, False)
 
-from clip_client import Client
+blank_clip_embedding = None
+blank_bert_embedding = None
 
+async def do_run(runtime_args, text_emb_clip):
+    global blank_clip_embedding, blank_bert_embedding
```

Comment:
why do we want this?

Reply:
`blank_clip_embedding, blank_bert_embedding` these embeddings constant always same for our case.
So I created those on the first time and re-used later calls.

Reply:
why not use cache for re-using constant embeddings?

Reply:
how should I cache it? `@lru_cache` decorator?